docs/user-guide/ze-profiles.md

```diff
@@ -3,13 +3,22 @@
 After you install Zowe Explorer, you need to have a Zowe Explorer profile to use all functions of the extension.
 
 :::info
-You can continue using Zowe V1 profiles with Zowe Explorer V2.
+You can continue using Zowe V1 profiles with Zowe Explorer V2. See [Working with Zowe CLI V1 profiles](#working-with-zowe-cli-v1-profiles) for more information.
 :::
-## Configuring team profiles
 
-Zowe CLI team profiles simplify profile management by letting you to edit, store, and share mainframe configuration details in one location. You can use a text editor or an IDE to populate configuration files with connection details for your mainframe services. By default, your team configuration file is located in the `.zowe home` folder, whereas the project-level configuration file is located in the main directory of your project. You can create profiles that you use globally, given that the names of the globally-used profiles are different from your other profile names.
+## Configuring Zowe V2 profiles
 
-**Note**: A project context takes precedence over global configuration.
+Zowe V2 uses *team profiles* to simplify profile management by letting you edit, store, and share mainframe connection details in one location, a configuration file.
+
+You can use a text editor or an IDE to populate configuration files with the connection information for your mainframe services. By default, your *global* team configuration file is located in the `.zowe home` folder, whereas the *project* configuration file is located in the main directory of your project.
+
+You can create profiles that you use globally, given that the names of the globally-used profiles are different from your other profile names.
+
+:::note
+
+A project context takes precedence over global configuration.
```

Comment:
It may be clearer to say "Project configuration takes precedence over global configuration.", just because the previous paragraph uses the term "configurations" for both project and global configs.

You could include a link to https://docs.zowe.org/stable/user-guide/cli-using-understand-profiles-configs#how-configuration-files-and-profiles-work-together to clarify what we mean by precedence.